### ISSUE TITLE
Use rounded rectangles to display abnormally

### ISSUE THREAD

**Issue by huang-weilong** [dnfield/lottie-flutter#3]
The rounded rectangle is used in AE effects, which will not display properly in flutter. What is the reason?

**Comment by dnfield** [comment]
Can you provide the lottie file that reproduces this?  Does it work properly in android or web?

**Comment by huang-weilong** [comment]
In the image displayed by this json file, the left side is a rounded rectangle and the right side is a square. It can be displayed normally on the web, but in the flutter project, the rounded rectangle will be distorted.

```
{
  "v": "5.2.1",
  "fr": 25,
  "ip": 0,
  "op": 50,
  "w": 1280,
  "h": 720,
  "nm": "Synthesis 2",
  "ddd": 0,
  "assets": [],
  "layers": [{
    "ddd": 0,
    "ind": 1,
    "ty": 4,
    "nm": "Shape 1",
    "sr": 1,
    "ks": {
      "o": {
        "a": 0,
        "k": 100,
        "ix": 11
      },
      "r": {
        "a": 0,
        "k": 0,
        "ix": 10
      },
      "p": {
        "a": 0,
        "k": [640, 360, 0],
        "ix": 2
      },
      "a": {
        "a": 0,
        "k": [0, 0, 0],
        "ix": 1
      },
      "s": {
        "a": 0,
        "k": [100, 100, 100],
        "ix": 6
      }
    },
    "ao": 0,
    "shapes": [{
      "ty": "gr",
      "it": [{
        "ty": "rc",
        "d": 1,
        "s": {
          "a": 0,
          "k": [252, 252],
          "ix": 2
        },
        "p": {
          "a": 0,
          "k": [0, 0],
          "ix": 3
        },
        "r": {
          "a": 0,
          "k": 0,
          "ix": 4
        },
        "nm": "Rectangular path 1",
        "mn": "ADBE Vector Shape - Rect",
        "hd": false
      }, {
        "ty": "st",
        "c": {
          "a": 0,
          "k": [1, 1, 1, 1],
          "ix": 3
        },
        "o": {
          "a": 0,
          "k": 100,
          "ix": 4
        },
        "w": {
          "a": 0,
          "k": 0,
          "ix": 5
        },
        "lc": 1,
        "lj": 1,
        "ml": 4,
        "ml2": {
          "a": 0,
          "k": 4,
          "ix": 8
        },
        "nm": "Stroke 1",
        "mn": "ADBE Vector Graphic - Stroke",
        "hd": false
      }, {
        "ty": "fl",
        "c": {
          "a": 0,
          "k": [0, 0, 0, 1],
          "ix": 4
        },
        "o": {
          "a": 0,
          "k": 100,
          "ix": 5
        },
        "r": 1,
        "nm": "Fill 1",
        "mn": "ADBE Vector Graphic - Fill",
        "hd": false
      }, {
        "ty": "tr",
        "p": {
          "a": 0,
          "k": [194, 12],
          "ix": 2
        },
        "a": {
          "a": 0,
          "k": [0, 0],
          "ix": 1
        },
        "s": {
          "a": 0,
          "k": [100, 100],
          "ix": 3
        },
        "r": {
          "a": 0,
          "k": 0,
          "ix": 6
        },
        "o": {
          "a": 0,
          "k": 100,
          "ix": 7
        },
        "sk": {
          "a": 0,
          "k": 0,
          "ix": 4
        },
        "sa": {
          "a": 0,
          "k": 0,
          "ix": 5
        },
        "nm": "Convert"
      }],
      "nm": "rectangle 2",
      "np": 3,
      "cix": 2,
      "ix": 1,
      "mn": "ADBE Vector Group",
      "hd": false
    }, {
      "ty": "gr",
      "it": [{
        "ty": "rc",
        "d": 1,
        "s": {
          "a": 0,
          "k": [264, 264],
          "ix": 2
        },
        "p": {
          "a": 0,
          "k": [0, 0],
          "ix": 3
        },
        "r": {
          "a": 0,
          "k": 20,
          "ix": 4
        },
        "nm": "Rectangular path 1",
        "mn": "ADBE Vector Shape - Rect",
        "hd": false
      }, {
        "ty": "st",
        "c": {
          "a": 0,
          "k": [1, 1, 1, 1],
          "ix": 3
        },
        "o": {
          "a": 0,
          "k": 100,
          "ix": 4
        },
        "w": {
          "a": 0,
          "k": 0,
          "ix": 5
        },
        "lc": 1,
        "lj": 1,
        "ml": 4,
        "ml2": {
          "a": 0,
          "k": 4,
          "ix": 8
        },
        "nm": "Stroke 1",
        "mn": "ADBE Vector Graphic - Stroke",
        "hd": false
      }, {
        "ty": "fl",
        "c": {
          "a": 0,
          "k": [0, 0, 0, 1],
          "ix": 4
        },
        "o": {
          "a": 0,
          "k": 100,
          "ix": 5
        },
        "r": 1,
        "nm": "Fill 1",
        "mn": "ADBE Vector Graphic - Fill",
        "hd": false
      }, {
        "ty": "tr",
        "p": {
          "a": 0,
          "k": [-298, 8],
          "ix": 2
        },
        "a": {
          "a": 0,
          "k": [0, 0],
          "ix": 1
        },
        "s": {
          "a": 0,
          "k": [100, 100],
          "ix": 3
        },
        "r": {
          "a": 0,
          "k": 0,
          "ix": 6
        },
        "o": {
          "a": 0,
          "k": 100,
          "ix": 7
        },
        "sk": {
          "a": 0,
          "k": 0,
          "ix": 4
        },
        "sa": {
          "a": 0,
          "k": 0,
          "ix": 5
        },
        "nm": "Convert"
      }],
      "nm": "rectangle 1",
      "np": 3,
      "cix": 2,
      "ix": 2,
      "mn": "ADBE Vector Group",
      "hd": false
    }],
    "ip": 0,
    "op": 50,
    "st": 0,
    "bm": 0
  }],
  "markers": []
}
```

**Comment by dnfield** [comment]
Thanks for the simple repro!

Fixed in e23a26d
